src/reactapp/src/components/paypalExpress/hooks/usePaypalExpressCartContext.js

```diff
@@ -0,0 +1,36 @@
+import { useContext, useMemo } from 'react';
+import _get from 'lodash.get';
+import CartContext from '../../../context/Cart/CartContext';
+import AppContext from '../../../context/App/AppContext';
+
+export default function usePaypalExpressCartContext() {
+  const [cartData] = useContext(CartContext);
+  const [, appActions] = useContext(AppContext);
```

Comment:
We should only deal with `CartContext` here. We need another hook like `usePaypalExpressAppContext`  to deal with `AppContext`